src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -20,7 +20,7 @@ def wrong_vmsize_argument_exception_handler(ex):
                     in format(ex)):
             raise CLIError('Invalid VM size. Example for Valid values: '
                            'For C family (C0, C1, C2, C3, C4, C5, C6), '
-                           'for P family (P1, P2, P3, P4)')
+                           'for P family (P1, P2, P3, P4, P5)')
```

Comment:
It won't hurt to store these values in one or two arrays. and use them in get_enum_type, help text and errors like this. That way if the SKUs are updated, it would be easier to keep this information up to date.